flake.nix

```diff
@@ -5,7 +5,7 @@
     #   - https://discourse.nixos.org/t/differences-between-nix-channels/13998
     # How to update the revision
     #   - `nix flake update --commit-lock-file` # https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-flake-update.html
-    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    nixpkgs.url = "github:kachick/nixpkgs/darwin-dprint-0.41";
```

Comment:
Using forked https://github.com/kachick/nixpkgs/pull/1 fixed the build failure https://github.com/kachick/dotfiles/actions/runs/6523617936/job/17714429877
I sent the PR for upstream as https://github.com/NixOS/nixpkgs/pull/261200